AllReadyApp/Web-App/AllReady/Views/Campaign/Index.cshtml

```diff
@@ -43,8 +44,14 @@
                         <i class="fa" data-bind="css: { 'fa-square-o': !campaigns.showOld(), 'fa-check-square-o': campaigns.showOld }"></i>
                         Include Completed Campaigns
                     </button>
-                    <span data-bind="text: '(' + total + ')' " ></span>
+                    <span data-bind="text: '(' + total + ')' "></span>
                 </div>
+                @if (await userAuthorizationService.IsCampaignManager())

```

Comment:
instead of injecting `@inject IUserAuthorizationService userAuthorizationService` at then calling:
```
@if (await userAuthorizationService.IsCampaignManager())
```

here, can you instead inject `IUserAuthorizationService` into the controller, add an `IsCampaignManager` property to the model and update the model's property instead of doing this in the view?

Call me old school, but injecting services into Views and then executing conditional statements based on those services (especially a service like this that incurs a db hit, and or cache hit) is work that should be done by the controller and/or the handler building up the ViewModel.